src/app/application.cpp

```diff
@@ -147,10 +147,8 @@ Application::Application(const QString &id, int &argc, char **argv)
 #if !defined(DISABLE_GUI)
     setAttribute(Qt::AA_UseHighDpiPixmaps, true);  // opt-in to the high DPI pixmap support
     setQuitOnLastWindowClosed(false);
-#if QT_VERSION >= QT_VERSION_CHECK(5, 7, 0)
     setDesktopFileName("org.qbittorrent.qBittorrent");
```

Comment:
appveyor failed due to this line, is appveyor using Qt 5.9 already?

Reply:
>appveyor failed due to this line, is appveyor using Qt 5.9 already?

Damn it. The cached files are probably qt 5.6.3.
I'll need to update this later.

Reply:
Fixed. I uploaded 5.9.7 for appveyor.